content/doc/book/blueocean/dashboard.adoc

```diff
@@ -21,27 +21,77 @@ image:blueocean/dashboard/overview.png[Overview of the Dashboard, role=center]
 
 == Navigation Bar
 
+The Dashboard includes the <<getting-started#navigation-bar, standard navigation bar>>
+at the top, with it's local navigation bar below that.
+The local navigation bar includes only one button:
+
+* *New Pipeline* - Open the <<creating-pipelines, Pipeline Creation workflow>>
 
 == Favorites
 
+The "Favorites" list is displayed above the regular Pipelines list.
+The "Favorites" list shows the status and key other details about the latest Run
+for branches or pull requests that include items that are important to the current user.
+Blue Ocean automatically adds branches or pull requests to this list when a they contain a
+Run that has changes authored by the current user.
+Users can also manually remove items from their favorites by clicking on the solid star in this list.
 
 == Pipelines
 
+The "Piplines" list show the overall health and status of each Pipeline in this Jenkins instance.
+// More to come
+
+
 
 [[pipeline-health]]
 === Health Icons
 
-// TODO: Describe health Icons
+Blue Ocean represents the overall health of a Pipeline or Pipeline branch using
+weather icons, which change depending on the number of recent builds that have Passed.
+Health icons on the dashboard represent overall Pipeline health.
+Health icons in the Activity Branches view represent health for each branch.
```

Comment:
Is "Activity Branches" enough of an independent concept or UI element that it should be quoted, similar to how you've quoted "Pull Requests" and "Merge Requests"?